aws/image/Makefile

```diff
@@ -19,6 +19,7 @@ $(IMAGE_FILE): setopts $(BINARIES) $(FILES)
 		-var region=${AWS_REGION} \
 		-var instance_type=${INSTANCE_TYPE} \
 		-var volume_size=${VOLUME_SIZE} \
+		-var config_script_src=$(ROOT_DIR)podvm/qcow2 \
```

Comment:
nit: I'd add "/" although not really needed

Reply:
Hmm, since the ROOT_DIR definition already has "/" I didn't add it. Although no harm in adding.